test/e2e-helm/e2e_helm_project_on_cluster_test.go

```diff
@@ -0,0 +1,282 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_helm_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo" //nolint:golint
+	. "github.com/onsi/gomega" //nolint:golint
+	kbtestutils "sigs.k8s.io/kubebuilder/test/e2e/utils"
+
+	testutils "github.com/operator-framework/operator-sdk/test/internal"
+)
+
+var _ = Describe("Running Helm projects", func() {
+	var controllerPodName string
+	const curlImage = "curlimages/curl:latest"
+
+	Context("built with operator-sdk", func() {
+		BeforeEach(func() {
+			By("Ensuring that the namespace is not created")
+			verifyControllerUp := func() error {
+				_, err := tc.Kubectl.Command("get", "namespace", tc.Kubectl.Namespace)
+				if strings.Contains(err.Error(), "(NotFound): namespaces") {
+					return err
+				}
+				return nil
+			}
+			Eventually(verifyControllerUp, time.Minute, time.Second).ShouldNot(Succeed())
+		})
+		AfterEach(func() {
+			By("Deleting Curl Pod created")
+			_, _ = tc.Kubectl.Delete(true, "pod", "curl")
+
+			By("Deleting CR instances created")
+			sampleFile := filepath.Join("config", "samples",
+				fmt.Sprintf("%s_%s_%s.yaml", tc.Group, tc.Version, strings.ToLower(tc.Kind)))
+			_, _ = tc.Kubectl.Delete(false, "-f", sampleFile)
+
+			By("Cleaning up permissions")
+			_, _ = tc.Kubectl.Command("delete", "clusterrolebinding",
+				fmt.Sprintf("metrics-%s", tc.TestSuffix))
+
+			By("Undeploy project")
+			_ = tc.Make("undeploy")
+		})
+
+		It("Should generate a runnable project on Cluster", func() {
```

Comment:
Nit: it seems like in all of the ginkgo examples the `Describe`, `It`, `Context`, `By`, etc. strings all start with lowercase words.

```suggestion
		It("should run correctly in a cluster", func() {
```